### PR TITLE
Added server file

### PR DESCRIPTION
**Pull request by zackstephens**

### CODE REVIEW

File: src/main/java/ParkingGarage/Client.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+        try {
+            // Create inputStream, assign user given port number
+            Scanner sc = new Scanner(System.in);
+            System.out.print("Enter the port number: <12345> ");
+            int port = sc.nextInt();
+
+            // Take server address from user 
+            System.out.print("Enter server address: ");
+            String host = sc.next();
```
Comment (aadiego) [inline src/main/java/ParkingGarage/Client.java:19]: Not that it matters, but is it weird we're asking for `port` first and then the `server address`?

File: src/main/java/ParkingGarage/Server.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        }
+    }
+
+}
+
+// maybe we put these as there own files?
+// or all in one message handler file?
+private void loginMessageFacade(Message message) {
+
+}
```
Comment (aadiego) [inline src/main/java/ParkingGarage/Server.java:93]: Ya, I agree that we should probably put these in their own files when we build them

File: src/main/java/ParkingGarage/Message.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    public String getStatus() {
+        return status;
+    }
+
+    public String getContent() {
+        return content;
+    }
+
+    // dont know if we need setters if
+    // we just create new messages for each response
```
Comment (aadiego) [inline src/main/java/ParkingGarage/Message.java:35]: Ya, I think we're going to probably want the messages to be immutable.

**Review by aadiego — APPROVED** [review]
Looks like a good base to start the rest of the project on.

File: src/main/java/ParkingGarage/Client.java (hunk 1 above)
Comment (zackstephens) [reply src/main/java/ParkingGarage/Client.java:19]: We can totally switch it

**Pull request merged**